File: 프로그래머스/0/120908. 문자열안에 문자열/문자열안에 문자열.py
def solution(str1, str2):
    return 1 if str2 in str1 else 2

# str.find("abc") : 찾고자하는 문자열이 존재할 경우, 시작 인덱스 값을 리턴 (없을 경우 -1 리턴)
# in, not in : 결과값이 있으면 True, 없으면 False 반환

# 다른 풀이
return 1 + int(str2 not in str1)

# 다른 풀이
if str1.count(str2):
    return 1
else:
    return 2

# 다른 풀이
answer = str1.split(str2)
if len(answer) == 1:
    return 2
else :
    return 1

File: 프로그래머스/0/120908. 문자열안에 문자열/문자열안에 문자열.java
class Solution {
    public int solution(String str1, String str2) {
        return str1.contains(str2) ? 1 : 2;
    }
}

// 다른 풀이
int answer = 0;
if(str1.indexOf(str2) != -1){
    answer = 1;
}
else answer =2;
return answer;
